@xen-orchestra/fs/src/s3.js

```diff
@@ -34,6 +34,8 @@ export default class S3Handler extends RemoteHandlerAbstract {
   }
 
   async _outputStream(input, path, { checksum }) {
+    const MAX_OBJECT_SIZE = 5 * 1024 * 1024 * 1024 * 1024
+    const FRAGMENT_SIZE = Math.ceil(MAX_OBJECT_SIZE / 10000)
```

Comment:
Why not define these constants out of the function?

Also, it would be nice to have a link to the doc explaining them.